src/output/RepresenterRewriteOutput.ts

```diff
@@ -29,7 +31,18 @@ export class RepresenterRewriteOutput implements Output {
     getProcessLogger().log(JSON.stringify(this.representation, undefined, 2))
     const normalized = normalizeRepresentation(this.representation)
 
-    const representation = generate(normalized.representation, {})
+
+    const quotesFormatGenerator = Object.assign({}, GENERATOR, {
+      Literal: function (node: LiteralNode, state: State) {
+        const { type, value } = node
+        const quote = `\``;
```

Comment:
Could this also be simplified to:

```suggestion
        const quote = '`';
```